docs/installation/linux/debian.md

```diff
@@ -132,7 +132,7 @@ Before installing Docker, make sure you have set your `APT` repository correctly
 
 5. Start the `docker` daemon.
 
-        $ sudo service docker start
+        $ sudo systemctl start docker
```

Comment:
Ah, yes, we may need a solution for this; (versions of Debian using systemd, versus older versions that don't). Doesnt Debian have a wrapper to make `service ..` also work on systemd?